newIDE/app/src/AssetStore/AssetPackInstallDialog.js

```diff
@@ -38,7 +38,7 @@ import { type InstallAssetOutput } from './InstallAsset';
 
 // We limit the number of assets that can be installed at once to avoid
 // timeouts especially with premium packs.
-const MAX_ASSETS_TO_INSTALL = 100;
+const MAX_ASSETS_TO_INSTALL = 200;
```

Comment:
this will match the value in `AssetsList` of 200, meaning that the button will always work and the warning never appear. So either reduce this one slightly, or increase the one in `AssetsList`